test/integration/targets/rabbitmq_basic_publish/tasks/ubuntu.yml

```diff
@@ -0,0 +1,155 @@
+- name: Install pika and requests
+  pip:
+    name: pika,requests
+    state: latest
```

Comment:
Constraints are needed for this pip install to avoid issues on Python 2.6.

Here's an example of the `extra_args` needed:

https://github.com/ansible/ansible/blob/14da4d66f23fe245a3ce652d3c56d93949937220/test/integration/targets/setup_docker/tasks/main.yml#L16-L19

Reply:
Also, use present for `requests`, not latest.

Reply:
Thanks for that.  The tests are targeted against Ubuntu 1604 as it depends on setup_rabbit.  1604 I believe has 2.7, so, not encountered any 2.6 issues in testing.

Changed requests as requested.